Dockerfile.ci

```diff
@@ -0,0 +1,37 @@
+#syntax=docker/dockerfile:1.4.0
+ARG CL_VERSION=develop-8
```

Comment:
We released 7.1.0, what will the tag look like now?

Reply:
It should be `v7.1.0`, but I wouldn't change it until https://github.com/aica-technology/control-libraries/actions/runs/5657706139 is finished anyway so that it re-triggers the CI on PR update